modules/tinymce/src/core/main/ts/file/Conversions.ts

```diff
@@ -27,9 +27,9 @@ const parseDataUri = (uri: string): Optional<DataUriResult> => {
   const [ type, ...rest ] = uri.split(',');
   const data = rest.join(',');
 
-  const matches = /data:([^/]+\/[^;]+)(;.+)?/.exec(type);
+  const matches = /data:([^/]+\/[^;]+)([^=]+=[^;]+)*(;.+)?/.exec(type);
   if (matches) {
-    const base64Encoded = matches[2] === ';base64';
+    const base64Encoded = matches[3] === ';base64';
```

Comment:
what about the case where there aren't any optional attributes? don't think we'd actually run into that, but since this is a third-party library it might be nice to account for both cases

Reply:
Should still work fine:
```
$ node
> test = 'data:image/gif;base64,R0ÖlGODdh'.split(',')[0]
'data:image/gif;base64'
> /data:([^/]+\/[^;]+)([^=]+=[^;]+)*(;.+)?/.exec(test)
[
  'data:image/gif;base64',
  'image/gif',
  undefined,
  ';base64',
  index: 0,
  input: 'data:image/gif;base64',
  groups: undefined
]
```